ui-tests/src/main/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -290,13 +312,13 @@ public void clickChangeDate (){
 		clickOn(CHANGE_DATE);
 	}
 
-	public void enterDate(){
+	public void enterDate() {
 		Calendar currentDay = Calendar.getInstance();
 	 	Integer dayOfMoth = currentDay.get(Calendar.DAY_OF_MONTH);
-	 	A: for(int j = 6; j > 0; j--) {
+	 	A: for(int j = 6; j > 0; j--) 

```

Comment:
Our convention is to have curly braces around statements inside if clauses, can you please add them back?